packages/instant/src/redux/store.ts

```diff
@@ -3,4 +3,5 @@ import { createStore, Store as ReduxStore } from 'redux';
 
 import { reducer, State } from './reducer';
 
-export const store: ReduxStore<State> = createStore(reducer);
+const reduxDevTools = (window as any).__REDUX_DEVTOOLS_EXTENSION__;
```

Comment:
For using redux dev tools in chrome.  I couldn't find a pattern for this in the website code, so went with this `(window as any)` hack

Reply:
I like using the package and having it be dev only, fixed in 3f0d7c70bd156edc4039d08737e74e99c0b3a04e